pandas/io/excel.py

```diff
@@ -1331,8 +1331,15 @@ def write_cells(self, cells, sheet_name=None, startrow=0, startcol=0,
             self.sheets[sheet_name] = wks
 
         if freeze_panes is not None:
-            wks.freeze_panes = wks.cell(row=freeze_panes[0] + 1,
-                                        column=freeze_panes[1] + 1)
+            if (
```

Comment:
@jeffcarey no preference. would be nice to have them be the *same* (could be a PR if you want). or even could do here. if you want.